### DISCUSSION TITLE
Spreading the word about JavaFX using Scene Builder

### DISCUSSION THREAD

**Discussion by Oliver-Loeffler** [gluonhq/scenebuilder#484] [Ideas]
All the good stuff about JavaFX is somehow distributed along the WWW but there no good curated point of entry.
Well now http://www.jfx-central.com has appeared. 

Would you think its a good idea to add a reference to http://www.jfx-central.com to the SB help menu? Kind of a community link?
It's also thinkable that we improve the welcome screen to directly provide some community links.

There could be a 2nd stage/tab on the Welcome dialog where we could add links to community content.
Links to openjfx.io, to jfx-central, to Gluon, Foojay, Reddit. 

Thoughts and ideas?

**Comment by dlemmermann** [comment]
JFXCentral could also provide a data feed with available libraries, blogs, etc.... we could also mark all libs that offer controls that can be easily imported into SceneBuilder. JFXCentral also knows their maven coordinates, so import could be seamless.

**Comment by Oliver-Loeffler** [comment]
@johanvos You have mentiond JFXCentral in todays talk. How about referencing it in Scene Builders help menu?

**Reply by johanvos** [reply]
I am ok with that.

**Reply by Oliver-Loeffler** [reply]
Cool, there's now a PR #570 - if you like, I'd be glad if you could review this one.
